src/aspire/source/__init__.py

```diff
@@ -8,7 +8,7 @@
     OrientedSource,
 )
 from aspire.source.relion import RelionSource
-from aspire.source.simulation import Simulation
+from aspire.source.simulation import LegacySimulation, Simulation
```

Comment:
Considering we only really want to use this for testing, can we call it `_LegacySimulation` to signal that it's not part of the public API?

Reply:
Sure, sounds good. Done!